pkg/config/property.go

```diff
@@ -34,13 +35,18 @@ const (
 
 	workspacePVCStorageClassName = "che.workspace_pvc.storage_class.name"
 
-	pluginArtifactsBrokerImage        = "che.workspace.plugin_broker.artifacts.image"
+	pluginArtifactsBrokerImage        = "controller.plugin_artifacts_broker.image"
 	defaultPluginArtifactsBrokerImage = "quay.io/eclipse/che-plugin-artifacts-broker:v3.1.0"
 
 	// routingClass defines the default routing class that should be used if user does not specify it explicitly
-	routingClass        = "che.workspace.default_routing_class"
+	routingClass        = "devworkspace.default_routing_class"
 	defaultRoutingClass = "basic"
 
+	// routingSuffix is the default domain for routes/ingresses created on the cluster. All
+	// routes/ingresses will be created with URL http(s)://<unique-to-workspace-part>.<routingSuffix>
+	routingSuffix        = "devworkspace.routing.cluster_host_suffix"
+	defaultRoutingSuffix = ""
+
 	workspaceIdleTimeout        = "che.workspace.idle_timeout"
```

Comment:
```suggestion
	workspaceIdleTimeout        = "devworkspace.idle_timeout"
```
?

Reply:
There are also "che.workspace.pvc.name" and "che.workspace_pvc.storage_class.name" which are not critical in WebTerminal case but still

Reply:
Fixed -- renamed to `devworkspace.pvc.*` and `devworkspace.idle_timeout`

There is also `	defaultTerminalDockerimageProperty = "che.workspace.default_dockerimage.redhat-developer.web-terminal"` in cmd_terminal.go. I can't come up with a good name for this one :D

Reply:
I have a good proposal neither. 
We can try to proceed with just `devworkspace.default_dockerimage.redhat-developer.web-terminal` and later we'll be able to migrate that if needed.

Reply:
Done, and squashed the PR commits into one.